connaisseur/tests/integration/LOCAL.md

```diff
@@ -0,0 +1,66 @@
+# Local integration test setup
+
+## Minikube
+
+To load the required docker images into the minikube docker daemon run
+
+```shell
+eval $(minikube docker-env)
+make docker
+```
+
+which will change the environment variables that specify the docker daemon to point to minikube instead of your system docker daemon.
+
+### Preparation for minikube with virtualbox driver
+
+Assuming you have a running minikube cluster using the virtualbox driver set as current kubernetes context you just need to have the alerting interface running.
+
+Run the docker container that serves as alerting endpoint and retrieve the IP address that it has on the `bridge` network:
+
+```shell
+docker run -d --name alerting-endpoint -p 56243:56243 docker.io/securesystemsengineering/alerting-endpoint:latest
+export ALERTING_ENDPOINT_IP=$(docker container inspect alerting-endpoint | jq -r .[].NetworkSettings.Networks.bridge.IPAddress)
+```
+
+### Preparation for minikube with docker driver
+
+Assuming you have a running minikube cluster using the docker driver set as current kubernetes context you need to have the alerting interface running and attach the container to the network that is used by minikube to facilitate communication. By default, this network is named `minikube`, so change the minikube network name to your specific name if you use a customized one:
```

Comment:
Funnily enough, minikube is no longer doing this on my machine 🤷‍♂️ Me confused

```shell
$ docker network list
NETWORK ID     NAME      DRIVER    SCOPE
bc53ec74fd03   bridge    bridge    local
49fede411976   host      host      local
ad8cfdd468b8   none      null      local
```

Reply:
So it does not Wörkst like a charm anymore? :cry:

Reply:
Well, then the vbox driver tutorial worked, so maybe distinction not by driver, but by "does it use a differnt docker network?", which is also more inclusive to other drivers I guess?

Reply:
Using any kind of docker driver should, in theory, use a docker bridge network, either a custom, newly created one or the default bridge network. We could add a network name lookup like

`docker container inspect minikube | jq -r '.[].NetworkSettings.Networks | to_entries | .[].key' `

then assuming the minikube container is actually named minikube?

Reply:
I guess that's a fair assumption, since apparently its the only valid container name: https://github.com/kubernetes/minikube/pull/6906